src/census_response.py

```diff
@@ -304,9 +305,11 @@ def __pd_process_poverty(cls):
             total_col = 'poverty_population_total'
             pct_df, pct_series = cls.__nest_percentages(poverty_df, total_col)
 
-            # create quantile bins
-            q_df = pct_df.apply(np.quantile, q=(0, 0.25, 0.5, 0.75, 1))
-            q_dict = q_df.to_dict(orient='list')
+            # create quantile bins using natural breaks algorithm. bin_count could be increased to > 4 if needed.
+            q_dict = calculate_natural_breaks_bins(pct_df, bin_count=4,
```

Comment:
Are 4 natural break bins equivalent to quartiles?

Reply:
Natural breaks compute clusters, so the bins will not be equal-sized (the vast majority of the time). So if I have 4 bins and 40 points, we could have bins of size 15, 5, 9, 11. So, "clusters" is probably the preferred nomenclature to use throughout the project. I didn't change it as I wasn't sure how that would affect encoding for firebase.

Reply:
I added quartiles bin back because I believe the client wants both. Changed the name of the natural breaks bin to "natural_breaks"